include/openssl/ml_kem.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifndef OSSL_MLKEM_CONSTANTS_H
+# define OSSL_MLKEM_CONSTANTS_H
```

Comment:
The define should be named as the file name and with OPENSSL prefix as in other public headers. `OPENSSL_ML_KEM_H` would be the right define.